prac_09/taxi_simulator.py

```diff
@@ -0,0 +1,55 @@
+from taxi import Taxi
+from silver_service_taxi import SilverServiceTaxi
+
+
+def main():
+    taxis = [Taxi("Prius", 100), SilverServiceTaxi("Limo", 100, 2), SilverServiceTaxi("Hummer", 200, 4)]
+    current_taxi = None
+    total_cost = 0.0
+
+    print("Let's drive!")
+    while True:
+        print("q)uit, c)hoose taxi, d)rive")
+        choice = input(">>> ").lower()
+        if choice == 'q':
+            print(f"Total trip cost: ${total_cost:.2f}")
+            print("Taxis are now:")
+            list_taxis(taxis)
+            break
+        elif choice == 'c':
+            current_taxi = choose_taxi(taxis)
+        elif choice == 'd':
+            if current_taxi:
+                total_cost += drive_taxi(current_taxi)
+            else:
+                print("You need to choose a taxi before you can drive")
+        else:
+            print("Invalid option")
+        print(f"Bill to date: ${total_cost:.2f}")
+
+
+def list_taxis(taxis):
```

Comment:
Some classes and methods in the code do not have documentation strings. It is recommended to add documentation strings for each class and public method that describe its functions and parameters.